tasty-query/shared/src/main/scala/tastyquery/reader/pickles/PickleReader.scala

```diff
@@ -31,9 +31,9 @@ private[pickles] class PickleReader {
   private var frozenSymbols: Boolean = false
 
   /** The map from created local symbols to the address of their info, until it gets read. */
-  private val localSymbolInfoRefs = mutable.AnyRefMap.empty[TermOrTypeSymbol, Int]
+  private val localSymbolInfoRefs = mutable.HashMap.empty[TermOrTypeSymbol, Int]
```

Comment:
`mutable.AnyRefMap` was deprecated in https://github.com/scala/scala/pull/10862 and now encourages use of `mutable.HashMap`